openapi.yaml

```diff
@@ -1246,48 +1262,27 @@ components:
                 description: Date this labor distribution becomes effective.
                 example: '1970-01-01'
               accountIndex:
-                type: object
-                properties:
-                  code:
-                    type: string
-                    nullable: false
-                    example: ABC123
+                type: string
+                nullable: false
+                example: ABC123
               fund:
-                type: object
-                properties:
-                  code:
-                    type: string
-                    nullable: true
+                type: string
+                nullable: true
```

Comment:
Examples of these? If you read the openapi doc from Swagger UI, you'll find some fields in the response body are "string", "0" which is a little confusing IMO. Can you add examples to those fields?